### PR TITLE
test(e2e): mint cip-68(222) handle

### PR DESCRIPTION
**Pull request by iadmytro**
# Context

Added minting CIP-68(222) handle during local network startup.

### CODE REVIEW

File: packages/e2e/local-network/scripts/mint-handles.sh [hunk 1]
```diff
@@ -51,3 +51,45 @@ cardano-cli transaction sign \
...
+      {"k": {"bytes": "76657273696f6e"}, "v": {"int": 0}},
+      {"k": {"bytes": "7465726d736f66757365"}, "v": {"bytes": "68747470733a2f2f63617264616e6f666f756e646174696f6e2e6f72672f656e2f7465726d732d616e642d636f6e646974696f6e732f"}},
+      {"k": {"bytes": "68616e646c65456e636f64696e67"}, "v": {"bytes": "7574662d38"}}
+    ]}},
+    {"k": {"bytes": "6e616d65"}, "v": {"bytes": "283130302968616e646c653638"}},
+    {"k": {"bytes": "696d616765"}, "v": {"bytes": "697066733a2f2f736f6d652d68617368"}},
+    {"k": {"bytes": "77656273697465"}, "v": {"bytes": "68747470733a2f2f63617264616e6f2e6f72672f"}},
+    {"k": {"bytes": "6465736372697074696f6e"}, "v": {"bytes": "5468652048616e646c65205374616e64617264"}},
+    {"k": {"bytes": "6175676d656e746174696f6e73"}, "v": {"list": []}}]},
+    {"int": 1}
```
Comment (mkazlauskas) [inline packages/e2e/local-network/scripts/mint-handles.sh:73]: Seems like we're missing the 3rd field:
```
The version number is at the second field of this constructor. The third field allows for arbitrary plutus data.
```
```
  ; Custom user defined plutus data.
    ; Setting data is optional, but the field is required
    ; and needs to be at least Unit/Void: #6.121([])
    extra = plutus_data
```

**Review by mkazlauskas — CHANGES_REQUESTED** [review]

File: packages/e2e/local-network/scripts/mint-handles.sh (hunk 1 above)
Comment (iadmytro) [reply packages/e2e/local-network/scripts/mint-handles.sh:110]: added in [e3bea15](https://github.com/input-output-hk/cardano-js-sdk/pull/807/commits/e3bea157c1d241db8de7a2505d33d5c1b579e153)

File: packages/e2e/local-network/scripts/mint-handles.sh [hunk 2]
```diff
@@ -70,3 +70,63 @@ cardano-cli transaction sign \
   --out-file handle-tx.signed
 
 cardano-cli transaction submit --testnet-magic 888 --tx-file handle-tx.signed
+
+sleep 10
```
Comment (mkazlauskas) [inline packages/e2e/local-network/scripts/mint-handles.sh:74]: Why is this needed and why 10? :sweat_smile:

**Review by mkazlauskas — APPROVED** [review]
Looks good! Just 1 question below

File: packages/e2e/local-network/scripts/mint-handles.sh (hunk 2 above)
Comment (iadmytro) [reply packages/e2e/local-network/scripts/mint-handles.sh:74]: it will take outdated utxo othervise

Comment (mkazlauskas) [reply packages/e2e/local-network/scripts/mint-handles.sh:74]: Sounds like we could see some flakiness due to this. Can we implement loop with sleep 1 that checks for state change instead?

**Review by mkazlauskas — APPROVED** [review]
Great work! :rocket:

**Review by rhyslbw — CHANGES_REQUESTED** [review]
Looks good @iadmytro. It would be better to reorder the commits so abc19d75a01cd32e73f3763221bc93c8fffb07d4 comes first

**Review by rhyslbw — APPROVED** [review]
Well done @iadmytro. Sorry I didn't realise [this line](https://github.com/input-output-hk/cardano-js-sdk/pull/807/files#diff-c245fa097162abd676a16c2a811730268a0542e4b802b5d06ea42271bd7d6f94L36) was existing, which was the basis for me requesting the commit reorder since it appeared to be added in the prior commit
